pyhive/presto.py

```diff
@@ -32,7 +33,16 @@
 paramstyle = 'pyformat'  # Python extended format codes, e.g. ...WHERE name=%(name)s
 
 _logger = logging.getLogger(__name__)
-_escaper = common.ParamEscaper()
+
```

Comment:
This is the upstream version for adding the 'timestamp' or 'date' prefix for Presto